sdk/core/Azure.Core/src/Pipeline/Internal/RequestActivityPolicy.cs

```diff
@@ -62,32 +62,34 @@ private async ValueTask ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPip
             using var scope = CreateDiagnosticScope(message);
 
             bool isActivitySourceEnabled = IsActivitySourceEnabled;
+            scope.SetDisplayName(message.Request.Method.Method);
 
-            scope.AddAttribute("http.method", message.Request.Method.Method);
-            scope.AddAttribute("http.url", _sanitizer.SanitizeUrl(message.Request.Uri.ToString()));
+            scope.AddAttribute(isActivitySourceEnabled ? "http.request.method" : "http.method", message.Request.Method.Method);
+            scope.AddAttribute(isActivitySourceEnabled ? "url.full" : "http.url", _sanitizer.SanitizeUrl(message.Request.Uri.ToString()));
             scope.AddAttribute(isActivitySourceEnabled ? "az.client_request_id": "requestId", message.Request.ClientRequestId);
+            if (message.RetryNumber > 0)
+            {
+                scope.AddIntegerAttribute("http.request.resend_count", message.RetryNumber);
+            }
 
             if (isActivitySourceEnabled && message.Request.Uri.Host is string host)
             {
-                scope.AddAttribute("net.peer.name", host);
-                int port = message.Request.Uri.Port;
-                if (port != 443)
-                {
-                    scope.AddIntegerAttribute("net.peer.port", port);
-                }
+                scope.AddAttribute("server.address", host);
+                scope.AddIntegerAttribute("server.port", message.Request.Uri.Port);
```

Comment:
The requirement to not add for default port was removed?

Reply:
will be removed very soon - https://github.com/open-telemetry/semantic-conventions/pull/459